src/openzaak/components/besluiten/tests/test_besluit_closed_zaak.py

```diff
@@ -0,0 +1,145 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2020 Dimpact
+
+from django.test import override_settings
+from django.utils import timezone
+from django.utils.translation import gettext as _
+
+from freezegun import freeze_time
+from rest_framework import status
+from rest_framework.test import APITestCase
+from vng_api_common.authorizations.models import Applicatie, Autorisatie
+from vng_api_common.constants import (
+    ComponentTypes,
+    VertrouwelijkheidsAanduiding,
+)
+from vng_api_common.models import JWTSecret
+from vng_api_common.tests import reverse, reverse_lazy
+
+from openzaak.components.autorisaties.tests.factories import CatalogusAutorisatieFactory
+from openzaak.components.besluiten.api.scopes import SCOPE_BESLUITEN_AANMAKEN
+from openzaak.components.besluiten.constants import VervalRedenen
+from openzaak.components.catalogi.tests.factories import (
+    BesluitTypeFactory,
+    InformatieObjectTypeFactory,
+)
+from openzaak.components.zaken.api.scopes import SCOPE_ZAKEN_GEFORCEERD_BIJWERKEN
+from openzaak.components.zaken.tests.factories import ZaakFactory
+from openzaak.tests.utils import JWTAuthMixin
+
+
+@freeze_time("2025-01-01T12:00:00")
+@override_settings(OPENZAAK_DOMAIN="testserver")
+class BesluitClosedZaakTests(JWTAuthMixin, APITestCase):
+    url = reverse_lazy("besluit-list")
+    max_vertrouwelijkheidaanduiding = VertrouwelijkheidsAanduiding.zeer_geheim
+
+    @classmethod
+    def setUpClass(cls):
+        APITestCase.setUpClass()
+
+        JWTSecret.objects.get_or_create(
+            identifier=cls.client_id, defaults={"secret": cls.secret}
+        )
+
+        cls.applicatie = Applicatie.objects.create(
+            client_ids=[cls.client_id],
+            label="for test",
+            heeft_alle_autorisaties=False,
+        )
```

Comment:
The `JWTAuthMixin` should do these things for you I think, probably if you first create the `besluittype` and then run `super().setUpTestData()`?